src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -143,6 +143,13 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Analyzers.NGram.MaxGram", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public int MaxGram { get; set; }
 
+        [Description("Managed allocations limit, which an index will stop and complete the current batch")]
```

Comment:
The description needs to mention that this is about memory _increase_ of managed allocations. I realized that after I saw the implementation where you calculate `diff`

Reply:
It isn't memory increase, we are allowing up to 512MB of allocations before closing the batch in this thread

Reply:
What do you think about the following description to make it clear that it's about allocations in a single batch:

"Managed allocations limit in an indexing batch after which the batch will complete and an index will continue by starting a new one"

Reply:
done